### ISSUE TITLE
Python 3.10 throws an error related to swintransformer

### ISSUE THREAD

**Issue by thistleknot** [clovaai/donut#234]
https://github.com/SwinTransformer/Swin-Transformer-Object-Detection/issues/214

whether 3.11 (rl9), 3.10 (rl9), 3.7 (3.7 ubu22 using conda/wsl)
AttributeError: 'SwinTransformer' object has no attribute 'pos_drop'

**Comment by poetaster** [comment]
I tried creating a custom env with conda, but it's not even possible to install the libraries as listed. any idea for a patch?

**Comment by poetaster** [comment]
duplicate of #207

The solution presented here ( comment out the one line) does indeed work (with timm=0.9.7)

**Comment by crackthedata** [comment]
@poetaster this fix worked for me :-)

**Comment by poetaster** [comment]
> @poetaster this fix worked for me :-)

Yeah, me too, just after some detours ;) Now if I could get VisionEncoder model chkpoints loading :)

**Comment by 7soonyounes** [comment]
i tried to unistall and install timm==0.5.4 but i still have this problem 
'SwinTransformer' object has no attribute 'pos_drop'
